api/agent/call.go

```diff
@@ -264,6 +264,10 @@ func (a *agent) GetCall(opts ...CallOpt) (Call, error) {
 		c.w = c.stderr
 	}
 
+	deadline := strfmt.DateTime(time.Now().Add(time.Duration(c.Call.Timeout) * time.Second)).String()
```

Comment:
there's a slight mismatch with the actual deadline that will be enforced from the context created inside of `agent.Submit` but I think the disparity will be very slight so this is ok (and ++4--0 is sweet)